collective/quickupload/browser/quickupload_settings.py

```diff
@@ -106,6 +106,18 @@ class IQuickUploadControlPanel(Interface):
         default=2,
         required=True)
 
+    object_unique_id = Bool(
+        title=_(u"title_object_unique_id", default=u"Choose unique file id"),
+        description=_(
+            u"description_object_unique_id",
+            default=u"Choose a unique id for each uploaded file."
+                    u"Checked: Your files will never be overridden and you'll "
+                    u"never get a 'file already exists' error message because "
+                    u"each file has its unique id."
+                    u"Non-Checked: The id will not be unique."),
```

Comment:
What does `The id will not be unique` mean? I think it is the job of an ID to be unique.
Maybe: `When not checked, the "Override by upload file" option is considered when a conflict happens." :grey_question: